accel-pppd/auth/auth_chap_md5.c

```diff
@@ -461,6 +463,8 @@ static void load_config(void)
 {
 	const char *opt;
 
+        config_lock();
```

Comment:
we've talked about config_lock() in every module, this will not be accepted.
also, please conform indent style